src/Tickets/Commerce/Ticket.php

```diff
@@ -270,7 +270,7 @@ public function get_ticket( $ticket_id ) {
 		}
 
 		/** @todo TribeCommerceLegacy: Remove the usage of Currency from Tribe Commerce totally, leave that behind. */
-		add_filter( 'tribe_get_option_ticket-commerce-currency-code', [ tribe( Legacy_Compat::class ), 'maybe_load_currency_code_from_tribe_commerce' ] );
+		add_filter( 'tribe_get_option_tickets-commerce-currency-code', [ tribe( Legacy_Compat::class ), 'maybe_load_currency_code_from_tribe_commerce' ] );
```

Comment:
@codingmusician the filter name was correct. We need to intercept calls to `tribe_get_option_ticket-commerce-currency-code` (Tribe Commerce calls)

Reply:
If we don't, we get Tribe Commerce settings overriding Tickets Commerce settings for currency